src/ray/core_worker/core_worker.cc

```diff
@@ -1263,6 +1263,11 @@ void CoreWorker::SpillOwnedObject(const ObjectID &object_id,
           RAY_LOG(ERROR) << "Failed to spill object " << object_id
                          << ", raylet unreachable or object could not be spilled.";
         }
+        // TODO(Clark): Provide spilled URL and spilled node ID to callback so it can
+        // added them to the reference.
+        /* const std::string &spilled_url = reply->object_url(); */
+        /* const NodeID &node_id = NodeID::FromBinary(reply->spilled_node_id()); */
+        /* callback(object_id, spilled_url, spilled_node_id); */
```

Comment:
Will remove this commented out code if we determine the current extra RPC route fine for this PR.